src/core/Makefile.am

```diff
@@ -81,6 +81,7 @@ libopenthread_a_SOURCES             = \
     thread/panid_query_server.cpp     \
     thread/thread_netif.cpp           \
     thread/thread_tlvs.cpp            \
+    utils/global_address.cpp          \
```

Comment:
Going to also need to add these files to the VS project files, under \etc\visual-studio. Specifically, the libopenthread.* and libopenthread_k.* files (4 in total).